### ISSUE TITLE
Scrollbar is not shown on High DPI

### ISSUE THREAD

**Issue by k-takata** [mintty/mintty#597]
Scrollbar is not shown on High DPI environment with the recent version (maybe after 2.5.0).

This is because [`WM_WINDOWPOSCHANGED`](https://github.com/mintty/mintty/blob/14f77a824bec98311d69b4fb04e28bef118bdcc4/src/winmain.c#L1524) is triggered after `CreateWindowEx()`, then `font_cs_reconfig()` is called, then `win_update_scrollbar()` is called, but [`term.show_scrollbar`](https://github.com/mintty/mintty/blob/14f77a824bec98311d69b4fb04e28bef118bdcc4/src/winmain.c#L1051) is false, so the scrollbar becomes hidden even if `cfg.scrollbar` is true.
`term.show_scrollbar` is set to true in `term_reset()` but it is not called yet when `WM_WINDOWPOSCHANGED` is triggered.

The following patch fixes the problem, but I'm not sure this is the right fix.

```diff
--- a/src/winmain.c
+++ b/src/winmain.c
@@ -2400,6 +2400,7 @@ main(int argc, char *argv[])
 #endif
 
   // Create initial window.
+  term.show_scrollbar = true;
   wnd = CreateWindowExW(cfg.scrollbar < 0 ? WS_EX_LEFTSCROLLBAR : 0,
                         wclass, wtitle,
                         window_style | (cfg.scrollbar ? WS_VSCROLL : 0),
```

**Comment by mintty** [comment]
Thanks for the analysis and patch already. The problem appeared in 2.4.3 and while the suggested patch is a workaround, there's some other sizing glitch in that update so I'll check other options.

**Comment by mintty** [comment]
Released 2.6.2.

**Comment by k-takata** [comment]
Thanks.
